docs/common_configurations.md

```diff
@@ -94,13 +94,8 @@ The `edpm_network_config_template` variable specifies the path to an ansible
 template that describes the networking configuration to be applied. The
```

Comment:
We need to change the text as it's not a path to the template any more. We should also mention here that the samples should be used inline in the CRs.

Reply:
Done